lib/metanorma/yaml2xml_collection.rb

```diff
@@ -0,0 +1,139 @@
+require "relaton-cli"
+# require "yaml"
+# require "tempfile"
+# require "relaton"
+# require "metanorma-cli"
+# require "nokogiri"
+
+module Metanorma
+  class Yaml2XmlColection
+    # @param file [String] YAML collection file path
+    def initialize(file)
+      @collection = YAML.load_file(file)
```

Comment:
I think this should be `Metanorma::Collection::Metadata`?

Reply:
What do you mean?

Reply:
The YAML provided is Metadata content of the Collection. I was just making a suggestion to the way to organize information.

Reply:
As I said before the two files `yaml2xml_collection.rb` and `collection_renderer.rb` made bay Nick. I just integrated these files into `metanoram` and add new command `collection` to `metanorma-cli`.
The `yaml2xml_collection.rb` is used to convert input file if it's in YAML format. The `collection_renderer.rb` render XML collection.
If we need code like this:
```ruby
collection = Metanorma::Collection.new
doc1 = Metanorma::Document.parse('doc1.xml')
doc2 = Metanorma::Document.parse('doc2.xml')
collection.metadata(title: "My collection")
collection << doc1
collection << doc2
collection.to_xml
```
then we need to make `Metanorma::Collection` and `Metanorma::Document`, and rewrite the existing code.

Reply:
Yes, we will need to do that. Nick's code was originally only intended for demonstration, but at least you now have the behavior correct to test a refactor 👍

Reply:
Do we have a description of YAML/XML collection?

Reply:
Nick's original ticket was the only description. Can you help elaborate on it? Thanks!